src/main/java/duke/exception/StorageFileNotFoundException.java

```diff
@@ -0,0 +1,16 @@
+package duke.exception;
+
+/**
+ * The StorageFileNotFoundException class represents a DukeException
+ * that is thrown when the data storage file cannot be found.
+ *
+ * @author Edric Yeo
+ */
+public class StorageFileNotFoundException extends DukeException {
```

Comment:
You could maybe name the classes that extend from DukeException in the format DukeStorageFileNotFoundException.